pkg/cli/common/util.go

```diff
@@ -70,3 +74,34 @@ func LoadFileIntoVar(filePath string, destContent *string) error {
 
 	return nil
 }
+
+// WriteConfigFile writes new content of the config file.
+// If the file does not exist, it is created at default location
+// TODO temporary solution until upstream https://github.com/spf13/viper/issues/433 is fixed
+func WriteConfigFile() error {
+	cf := viper.ConfigFileUsed()
+
+	if cf == "" {
+		fullname := ConfigFileName + "." + ConfigFileType
+		if dirname, err := os.Getwd(); err == nil {
+			cf = filepath.Join(dirname, ConfigHomeSubdir, fullname)
+		}
+		if dirname, err := os.UserHomeDir(); err == nil {
+			cf = filepath.Join(dirname, ConfigHomeSubdir, fullname)
```

Comment:
I would like to recommend storing the configuration file at `$HOME/.config` which I believe is the standard for user-specific configuration files (https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html)

Reply:
Well, we already have `ConfigHomeSubdir` defined, so I assume you want to get rid of that, right? Because this is not related to the PR which solves the problem of creating the file

Reply:
+1 to the `$HOME/.config` idea. It is related though, because you have to create the parent directory in order to create the config file.

Reply:
OK @stefannica @flaviodsr , so do you mean
- `$HOME/.config/fuseml.yaml` (that was my original intention) or
- `$HOME/.config/fuseml/config.yaml` (this actually has the installer, or at least had initially, maybe it is not using it any more)

Reply:
I like the second one better (`$HOME/.config/fuseml/config.yaml`).

As for the alternative, IMO it is not needed as you can just specify the config using env variables, I think it should just error if the config was not found/cannot be created.

Reply:
OK then, check the latest update